src/lib/sedlexing.ml

```diff
@@ -43,11 +43,24 @@ type lexbuf = {
   mutable marked_val: int;
 
   mutable finished: bool;
+
+  (* Compat with other lexers. *)
+  mutable lex_marked_p: Lexing.position;
+  mutable lex_start_p:  Lexing.position;
+  mutable lex_curr_p:   Lexing.position
 }
 
 let chunk_size = 512
 
-let empty_lexbuf = {
+let empty_position () = {
+  Lexing.
+    pos_fname = "";
+    pos_lnum = 1;
+    pos_bol = 0;
+    pos_cnum = 0;
+}
```

Comment:
These need to be freshly instantiated but you can use other constructs if you prefer.